docs/integrations/vector-databases/qdrant/overview.md

```diff
@@ -0,0 +1,11 @@
+---
+title: Overview - Qdrant
+description: This section of the Quix documentation covers how to integrate QDrant with Quix.
+search:
```

Comment:
This isn't required.

Reply:
It causes the page to rank higher in search, which is not required for this page. It's only used for very specific pages we want to boost.

Reply:
Also, I just notice you have two capitalizations of Qdrant - in the description you use "QDrant" which is not correct from brief check of their website.